src/introspection/introspection.ts

```diff
@@ -202,6 +202,23 @@ function markDeprecated(schema: SimplifiedIntrospectionWithIds): void {
   // which are deprecated.
 }
 
+function markInterfaceFields(schema: SimplifiedIntrospectionWithIds): void {
+  _.each(schema.types, type => {
+    if (type.kind === 'OBJECT') {
+      if (type.interfaces && type.interfaces.length > 0) {
+        // we have some interfaces for this object
+        // so we should delete fields that are presented in interface
+        // and also present in object
+        _.each(type.interfaces, oneInterface => {
+          _.each(oneInterface.type.fields, field => {
+            delete type.fields[field.name];
```

Comment:
You should delete fields only if fully equal to the interface's field.
Counterexample:
```gql
interface Foo {
  foo: String
}

type Bar implements Foo {
  foo(arg: Int): String
}
```

Reply:
Also now you can have type without fields please check how they are rendering both in graph UI and left the panel.

Reply:
Uff i didn't know that you can override field definition from interface inside concrete implementation.

Type without fields(as they are all same as in interface) is currently ok because i added list of interfaces that concrete type implements
![image](https://user-images.githubusercontent.com/5859040/70179459-628a4100-16de-11ea-9855-ab8afea92642.png)

Reply:
And for left panel it is kind of sad because if you click on Foo type it will show only interface that it implements instead of all fields which should be preferable as there is place for it.

![image](https://user-images.githubusercontent.com/5859040/70179785-ffe57500-16de-11ea-8bec-b10c6198c0d1.png)

Maybe better than deleting fields might be more sensible to add some flag to them and then when they are rendered into graph view they will be hidden

Reply:
Ok i have prototype where instead of deleting fields on object (because than we can't show them in docs on left side) i added flag `isSameAsOnInterface` on field definition and then when it is drawing type box is it reuse `canDisplayRow` which hides enums and scalars with option `showLeafFields` with similar logic for fields from interface.

![image](https://user-images.githubusercontent.com/5859040/70181450-31137480-16e2-11ea-9651-9034f247a141.png)

Reply:
And with example
```
interface Foo {
  foo: String
}

type Bar implements Foo {
  foo(arg: Int): String
}
```
Do you have idea how condition for checking if field on interface and type are same? Name should be same. args should have same maps with name? Is there anything else?

I checked you cant change type of returned thing. Nor you can change type inside arguments like this

Reply:
@matystl Here are the rules for implementing interfaces: https://graphql.github.io/graphql-spec/draft/#sel-HAHZhCFHABABiCp7I